install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -1,4 +1,10 @@
-You have successfully installed {{ title .Chart.Name }}.
+{{- if (and (.Values.preflight.enabled) (not (.Values.agent.enabled)) (not (.Values.config.enabled)) (not (.Values.operator.enabled))) }}
+    You have successfully completed the prefight check.
```

Comment:
I'm actually wondering about the phrasing here. The only thing we know was successful at this point in time is that the YAMLs were generated and injected into the cluster. 

Looking at the preflight check documentation, we actually suggest using another command to monitor the status of the prefilght check until it becomes successful. I wonder if those instructions would be appropriate to include in the snippet here?

Reply:
Will check it out and see what makes sense.